app/data.py

```diff
@@ -0,0 +1,39 @@
+from app.customer import Customer
+from app.car import Car
+from app.shop import Shop
+import json
+
+
+def process_json_data() -> list:
+    customers = []
+    shops = []
+
+    with open("app/config.json") as file:
+        data = json.load(file)

```

Comment:
Consider adding error handling for the file reading process. If the file does not exist or contains invalid JSON, this will raise an error. You can use a try-except block to handle exceptions like `FileNotFoundError` or `json.JSONDecodeError`.